client/src/ui/atoms/Checkbox.test.tsx

```diff
@@ -32,20 +32,24 @@ describe('Checkbox', (): void => {
     });
 
     it('should load with checked value if initialValue true', async (): Promise<void> => {
-        <Checkbox id="test" invalid={false} labelText="some label" />
-        const { container } = render(<Checkbox id="test" invalid={false} labelText="some label" initialValue={true}/>);
+        <Checkbox id="test" invalid={false} labelText="some label" />;
```

Comment:
Do we want to mandate semicolons?

Reply:
It's current part of the linting rules. This will have been changed by the `lint -fix` script. They're not required though so we could change this linting rule if we want.